[2025-10-23T20:51:13.017+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_branch_dop_operator_v3.should_run scheduled__2025-10-23T20:50:00+00:00 [queued]>
[2025-10-23T20:51:13.020+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_branch_dop_operator_v3.should_run scheduled__2025-10-23T20:50:00+00:00 [queued]>
[2025-10-23T20:51:13.021+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-10-23T20:51:13.025+0000] {taskinstance.py:1382} INFO - Executing <Task(_BranchPythonDecoratedOperator): should_run> on 2025-10-23 20:50:00+00:00
[2025-10-23T20:51:13.027+0000] {standard_task_runner.py:57} INFO - Started process 4427 to run task
[2025-10-23T20:51:13.028+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'example_branch_dop_operator_v3', 'should_run', 'scheduled__2025-10-23T20:50:00+00:00', '--job-id', '532', '--raw', '--subdir', '/home/airflow/.local/lib/python3.11/site-packages/airflow/example_dags/example_branch_python_dop_operator_3.py', '--cfg-path', '/tmp/tmp49svwjag']
[2025-10-23T20:51:13.029+0000] {standard_task_runner.py:85} INFO - Job 532: Subtask should_run
[2025-10-23T20:51:13.052+0000] {task_command.py:416} INFO - Running <TaskInstance: example_branch_dop_operator_v3.should_run scheduled__2025-10-23T20:50:00+00:00 [running]> on host 39b6b9dcf23a
[2025-10-23T20:51:13.089+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='example_branch_dop_operator_v3' AIRFLOW_CTX_TASK_ID='should_run' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T20:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-23T20:50:00+00:00'
[2025-10-23T20:51:13.091+0000] {logging_mixin.py:154} WARNING - /home/airflow/.local/lib/python3.11/site-packages/airflow/utils/context.py:314 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-10-23T20:51:13.091+0000] {logging_mixin.py:154} INFO - ------------- exec dttm = 2025-10-23T20:50:00+00:00 and minute = 50
[2025-10-23T20:51:13.092+0000] {python.py:194} INFO - Done. Returned value was: empty_task_1
[2025-10-23T20:51:13.092+0000] {python.py:227} INFO - Branch callable return empty_task_1
[2025-10-23T20:51:13.092+0000] {skipmixin.py:173} INFO - Following branch empty_task_1
[2025-10-23T20:51:13.100+0000] {skipmixin.py:239} INFO - Skipping tasks [('empty_task_2', -1)]
[2025-10-23T20:51:13.120+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_branch_dop_operator_v3, task_id=should_run, execution_date=20251023T205000, start_date=20251023T205113, end_date=20251023T205113
[2025-10-23T20:51:13.167+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-10-23T20:51:13.181+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
